pkgs/applications/virtualization/nvidia-docker/libnvc.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchFromGitHub, fetchurl, libelf, libcap, libseccomp }:
+
+with lib; let
+
+  nvidia-modprobe = fetchurl {
+    url = "https://github.com/NVIDIA/nvidia-modprobe/archive/396.51.tar.gz";
```

Comment:
Since this is a github url, you can use `fetchFromGitHub` instead.

Reply:
Done, I copy it manually now and chmod below.